doc/rules/function_notation/phpdoc_to_return_type.rst

```diff
@@ -30,6 +30,15 @@ Allowed types: ``bool``
 
 Default value: ``true``
 
+``union_types``
+~~~~~~~~~~~~~~~
+
+Fix also union types; turned on by default on PHP >= 8.0.0.
```

Comment:
I'm OK with the state of this PR.
with recent claim "test is contract", some folks can consider this change to be BC break and want to have it as opt-in instead opt-out.

Reply:
nope, feature is already there, we only allow to opt-out instead of force-enabling.